releasetasks/templates/firefox/routes.yml.tmpl

```diff
@@ -0,0 +1,23 @@
+# Routing Templates: releases v2!
+
+# version (simple, direct route)
+{% do release_routes.append('releases.v2.{app}.version.{version}.build{build_num}'
+        .format(app=product, version=version.replace('.', '_'), build_num=buildNumber)) %}
+# pushdate
+{% do release_routes.append('releases.v2.{app}.pushdate.{year}.{mm}.{dd}.{version}.build{build_num}'
+        .format(app=product, year=now.year, mm=now.month, dd=now.day,
+                version=version.replace('.', '_'), build_num=buildNumber)) %}
+# revision
+{% do release_routes.append('releases.v2.{app}.revision.{branch}.{revision}.{version}.build{build_num}'
+        .format(app=product, branch=branch, revision=revision,
+                version=version.replace('.', '_'), build_num=buildNumber)) %}
+# latest revision (convenience)
+{% do release_routes.append('releases.v2.{app}.revision.{branch}.latest.{version}.build{build_num}'
```

Comment:
Why does the "latest" route also have a version and build_num in it? I'd think the latest would just be of the form 'releases.v2.{app}.{branch}', and would point to the latest release of that app/branch (eg: the latest firefox on mozilla-release).